### PR TITLE
Fix for issue 322

### PR DESCRIPTION
**Pull request by llopezme-cisco**

### CODE REVIEW

**Comment by llopezme-cisco** [comment]
@tzarski0 can you take a look this one please.

File: defaults/sdwan.yaml [hunk 1]
```diff
@@ -316,3 +316,7 @@ defaults:
         aaa:
           auth_order:
             - local
+    centralized_policies.definitions:
```
Comment (tzarski0) [inline defaults/sdwan.yaml:319]: The correct structure should be:
```
    centralized_policies:
      definitions:
        data_policy:
          cflowd:
            protocol: ipv4
```

File: defaults/sdwan.yaml [hunk 2]
```diff
@@ -316,3 +316,7 @@ defaults:
...
+      data_policy:
+        cflowd:
+          protocol: "ipv4"
```
Comment (tzarski0) [inline defaults/sdwan.yaml:322]: we should also add defaults for tos: false and remarked_dscp: false (when you configure cflowd in UI, by default they are set to false)

Comment (tzarski0) [inline defaults/sdwan.yaml:322]: no need for quoted and we can replace "ipv4" with just ipv4

File: sdwan_centralized_policies.tf [hunk 3]
```diff
@@ -553,13 +553,13 @@ resource "sdwan_cflowd_policy_definition" "cflowd_policy_definition" {
...
-  sampling_interval     = each.value.sampling_interval
-  flow_refresh          = each.value.flow_refresh
-  protocol              = each.value.protocol
-  tos                   = each.value.tos
-  remarked_dscp         = each.value.remarked_dscp
+  active_flow_timeout   = try(each.value.active_flow_timeout, null)
+  inactive_flow_timeout = try(each.value.inactive_flow_timeout, null)
+  sampling_interval     = try(each.value.sampling_interval, null)
+  flow_refresh          = try(each.value.flow_refresh, null)
+  protocol              = try(each.value.protocol, local.centralized_policies.definitions.data_policy.cflowd.protocol)
```
Comment (tzarski0) [inline sdwan_centralized_policies.tf:560]: should be:
```
  protocol              = try(each.value.protocol, local.defaults.sdwan.centralized_policies.definitions.data_policy.cflowd.protocol)
```

File: sdwan_centralized_policies.tf [hunk 4]
```diff
@@ -553,13 +553,13 @@ resource "sdwan_cflowd_policy_definition" "cflowd_policy_definition" {
...
+  tos                   = try(each.value.tos, null)
```
Comment (tzarski0) [inline sdwan_centralized_policies.tf:561]: let's add default value here as described in previous comment

File: sdwan_centralized_policies.tf [hunk 5]
```diff
@@ -553,13 +553,13 @@ resource "sdwan_cflowd_policy_definition" "cflowd_policy_definition" {
...
+  remarked_dscp         = try(each.value.remarked_dscp, null)
```
Comment (tzarski0) [inline sdwan_centralized_policies.tf:562]: let's add default value here as described in previous comment

**Review by tzarski0 — CHANGES_REQUESTED** [review]
@llopezme-cisco looks good, but requires few minor changes. Could you please work on the update?

**Review by tzarski0 — APPROVED** [review]

**Review by tzarski0 — APPROVED** [review]

**Pull request merged**